packages/libs/react-ui/src/components/BrandLogo/index.tsx

```diff
@@ -1,12 +1,13 @@
-import { KadenaLogo } from './variants/Kadena';
-import { KadenaDevToolsLogo } from './variants/KadenaDevTools';
-import { KadenaDocsLogo } from './variants/KadenaDocs';
+import { KadenaLogo } from './svgs/Kadena';
+import { KadenaDevToolsLogo } from './svgs/KadenaDevTools';
+import { KadenaDocsLogo } from './svgs/KadenaDocs';
 
 import type { FC, SVGProps } from 'react';
 import React from 'react';
 
-export type LogoVariant = 'Kadena' | 'DevTools' | 'Docs';
-export const logoVariants: LogoVariant[] = ['Kadena', 'DevTools', 'Docs'];
+// eslint-disable-next-line @kadena-dev/typedef-var
```

Comment:
Do we need this? Works fine without it for me.

Reply:
This is just a minor change that allows us to add the logos to the constant and generates the types based on that array instead of having to edit both the type and array. It's not likely that these would get out of sync since they are right next to each other, but I just have a preference to make updates in as few places as possible to avoid accidental mistakes

Reply:
Yeah I know what it does, I meant the disable. My IDE seems fine with it, without the eslint-disable?

Reply:
I get a warning when running the lint script